src/main/java/nl/tudelft/ewi/devhub/server/web/resources/AccountResource.java

```diff
@@ -53,6 +53,16 @@
 		this.currentUser = currentUser;
 		this.users = users;
 	}
+	
+	private User verifyCanAccessPage(String netId) throws UnauthorizedException {
```

Comment:
Maybe we want to rename this method because it also provides the user to edit (admins can edit students). But I can't think of any alternative.